resources/functionMap.php

```diff
@@ -8075,7 +8075,7 @@
 'php_logo_guid' => ['string'],
 'php_sapi_name' => ['__benevolent<non-empty-string|false>'],
 'php_strip_whitespace' => ['string', 'file_name'=>'string'],
-'php_uname' => ['string', 'mode='=>'string'],
+'php_uname' => ['string', 'mode='=>'"a"|"s"|"n"|"r"|"v"|"m"'],
```

Comment:
I don't accept parameter type restrictions outside of bleeding edge. Because this would require users to also narrow down their PHPDocs when they're currently just passing `string` there.

So this change belongs to `resources/functionMap_bleedingEdge.php`.